### DISCUSSION TITLE
Can i use a RP2, or RP3, receiver with a powerbox System like the Mercury SR2

### DISCUSSION THREAD

**Discussion by danielcf1000** [ExpressLRS/ExpressLRS#3083] [Q&A]
This powerBox for example can be used with to futaba receivers that connect to it by S.bus, i dont know if is possible with a RP2-3 receiver do that and how if posible.

I m looking to a build with 11+ servos(Maxi Lift from seagull models) i m very happy with the radiomaster ER PWM receivers series but they only go up to 8 channels and i m not looking to use other brand, i seen there are some beta  FPV with 10 PWM channels, but emailed radiomaster and they have planned an 12*channel PWM receiver on the making, but if the above option connecting the receivers directly would be posible it would be better i guess?

**Comment by fzlrhmn** [comment]
I did this with 2 Radiomaster ER6 working as redundant receiver connected to Dualsky Gemini 3018 by using SBUS connection. It just works. The downside I think the failsafe will not work as intended when there is cable/connection issue between RX and the box. 

I am waiting my Radiomaster DBR4 order and plan to use that with Gemini connection. So it can have solid 4 connection. I am doing this because my field where I play has really high interference (it is active military airfield, even Jeti has crashed here multiple times). 
![IMG_0179](https://github.com/user-attachments/assets/7a473d82-3634-452d-9af5-fb6eaf3d63ee)

**Comment by danielcf1000** [comment]
Wow, you've given me an idea, please keep me posted, because you're tempting me to use a Gemini power board like yours but with RP4 diversity receivers!

@fzlrhmn

**Reply by fzlrhmn** [reply]
I intended to use RP4TD previously but the price is not that different from DBR4. So I choose DBR4 that has elrs gemini/gemx as well (So many Gemini words right there). I will posted it when the receiver arrive.
